generate.js

```diff
@@ -647,14 +678,17 @@ function getCompiledTemplate(templatePath) {
 }
 global.getCompiledTemplate = getCompiledTemplate;
 function doNothing() { }
-try {
-    // Kick everything off
-    parseAndLoadApis();
-}
-catch (error) {
-    console.error(error);
-    setTimeout(doNothing, 30000);
-    throw error;
+function catchAndReport(method) {
+    try {
+        method();
+    }
+    catch (error) {
+        console.error(error);
+        setTimeout(doNothing, 30000);
+        // throw(error);
+    }
 }
+// Kick everything off
+catchAndReport(parseAndLoadApis);
 setTimeout(doNothing, 5000);
```

Comment:
Btw i don't think we need these timeouts anymore

Reply:
I'm willing to discuss them separately, but I like/need them, and I don't have a good workaround.
(VS and NodeJS 10 don't get along properly)